packages/flutter_libphonenumber_android/android/build.gradle

```diff
@@ -2,14 +2,14 @@ group 'com.bottlepay.flutter_libphonenumber'
 version '1.0-SNAPSHOT'
 
 buildscript {
-    ext.kotlin_version = '1.6.10'
+    ext.kotlin_version = '1.8.21'
```

Comment:
Since we are already in here, we can set it as `1.8.22` as of June 23, 2023

Reply:
Happy to do this if it's still relevant.